deploy/roles/neo4j/handlers/main.yml

```diff
@@ -1,4 +1,4 @@
 # main.yml
 ---
 - name: restart neo4j
-  command: /opt/neo4j/neo4j-community-3.0.4/bin neo4j restart
+  command: "{{ neo4j_home_dir }}/bin/neo4j restart"
```

Comment:
@olivermel Now that we have the upstart script, the command should just be `restart neo4j` instead of `{{ neo4j_home_dir }}/bin/neo4j restart`